### DISCUSSION TITLE
进阶篇 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#329] [General]
# 进阶篇 | FixIt

探索 Hugo - FixIt 主题的的进阶使用。

https://fixit.lruihao.cn/zh-cn/docs/advanced/

**Comment by wztzt** [comment]
是

**Comment by Lruihao** [comment]
FixIt 经过近四年的开发仍旧处于活跃开发状态，如果你在使用主题的过程中觉得有哪一部分是需要进一步文档说明的，请留言告知以便完善文档❤️
